drivers/include/hdc1000.h

```diff
@@ -136,7 +139,32 @@ int hdc1000_trigger_conversion(const hdc1000_t *dev);
  * @return                  HDC1000_OK on success
  * @return                  HDC1000_BUSERR on I2C communication failures
  */
-int hdc1000_get_results(const hdc1000_t *dev, int16_t *temp, int16_t *hum);
+int hdc1000_get_results(hdc1000_t *dev, int16_t *temp, int16_t *hum);
+
+/**
+ * @brief   Check if cached values are outdated or not
+ *
+ * @param[in]  dev          device descriptor of sensor
+ *
+ * @return                  true if cached values are outdated
+ * @return                  false otherwise
+ */
+bool hdc1000_outdated(hdc1000_t *dev);
```

Comment:
As stated above, I'm against this extra functino. However, if you would argue explicitly **for** such a function, it should be at least inline and not part of the public API.